.pylintrc

```diff
@@ -25,10 +25,6 @@ ignore-patterns=
 # pygtk.require().
 #init-hook=
 
-# Use multiple processes to speed up Pylint. Specifying 0 will auto-detect the
-# number of processors available to use.
-jobs=0
```

Comment:
Why not keep `jobs` here, but set a value higher than 0?

Reply:
> Why not keep `jobs` here, but set a value higher than 0?

it's in the upstream file in `depot_tools`, we cannot patch it.

Reply:
You mentioned that we always pass an explicit -j. So the value here would only be used in some narrow cases, where it was breaking due to the 0. So if we set it to 56 here, there would be no change where we pass -j and we wouldn't see breakage where it is now. There are arguments for and against it, but wanted to point out the option.

Feel free to resolve this, if you think the code should stay as is, I'll approve the PR.

Reply:
Thanks. I think here is the best option is to remove this value, because `multiprocessing` *always* creates all processes even if pylint needs to check just one file. This takes some notable time (3-5 seconds on Windows), so in the case when nothing is passed, we need to fallback to 1 (not 32 or 56) to have fast presubmit checks.